blender_addons/io_md5_exporter/io_export_md5.py

```diff
@@ -797,16 +804,16 @@ def treat_bone(b, parent = None):
 
   if modelFilePath != None:
       #save all submeshes in the first mesh
-      if len(meshes)>1:
+      if len(meshes) > 1:
         for mesh in range (1, len(meshes)):
           for submesh in meshes[mesh].submeshes:
             submesh.bindtomesh(meshes[0])
+      createDirectoryForFileIfMissing(modelFilePath)
       try:
         file = open(modelFilePath, 'w')
-      except IOError:
-        errmsg = "IOError " #%s: %s" % (errno, strerror)
+      except:
+        pass

```

Comment:
pass would mean simply ignoring the exception without any logging which would be bad